Ratnesh/week 2 assignment 1 by ratnesh/count duplicate array.c

```diff
@@ -0,0 +1,31 @@
+#include <stdio.h>
+ 
+int main()
+{
+  int arr[10], i, j, Size, Count = 0;
+  
+  printf("\n  Number of elements in an array  :   ");
```

Comment:
"Comment can be-: Enter Number of elements in an array"

Reply:
who see comments like that

Reply:
that's right. but some time the user will get confuse so only i told.